src/collections/maple_tree/denseLeaf.rs

```diff
@@ -0,0 +1,236 @@
+use super::constants::D_CAPACITY;
+use super::utils::M;
+use std::fmt::Debug;
+use std::mem;
+
+#[derive(Debug)]
+pub struct DenseLeaf<V> {
+    value: [M<V>; D_CAPACITY],
```

Comment:
Now that it's stable, we should look at MaybeUninit too. But that's a future us problem.